website/docs/vision.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Vision
+sidebar_label: Vision
+sidebar_position: 2
+---
+
+## Charter
+
+**One Layer, All Storage.**
+
+## Tenets
```

Comment:
"Principles"?

Tenets is less to use.

Reply:
This word is derived from "hyper." I'm open to changing it.